tiflash/troubleshoot-tiflash.md

```diff
@@ -170,40 +170,23 @@ After deploying a TiFlash node and starting replication (by performing the ALTER
         }' <http://172.16.x.xxx:2379/pd/api/v1/config/rule>
     ```
 
-5. Check whether the connection between TiDB or PD and TiFlash is normal.
+5. Check whether TiDB has created `placement-rule` for tables.
 
-    Search the `flash_cluster_manager.log` file for the `ERROR` keyword.
-
-    - If no `ERROR` is found, the connection is normal. Go to the next step.
-    - If `ERROR` is found, the connection is abnormal. Perform the following check.
-
-      - Check whether the log records PD keywords.
-
-          If PD keywords are found, check whether `raft.pd_addr` in the TiFlash configuration file is valid. Specifically, run the `curl '{pd-addr}/pd/api/v1/config/rules'` command and check whether there is any output in 5s.
-
-      - Check whether the log records TiDB-related keywords.
-
-          If TiDB keywords are found, check whether `flash.tidb_status_addr` in the TiFlash configuration file is valid. Specifically, run the `curl '{tidb-status-addr}/tiflash/replica'` command and check whether there is any output in 5s.
-
-      - Check whether the nodes can ping through each other.
-
-    > **Note:**
-    >
-    >  If the problem persists, collect logs of the corresponding component for troubleshooting.
-
-6. Check whether `placement-rule` is created for tables.
-
-    Search the `flash_cluster_manager.log` file for the `Set placement rule … table-<table_id>-r` keyword.
+    Search the logs of TiDB DDL Owner and check whether TiDB has notified PD to add `placement-rule`. For non-partitioned tables, search `ConfigureTiFlashPDForTable`. For partitioned tables, search `ConfigureTiFlashPDForPartitions`.
 
     - If the keyword is found, go to the next step.
     - If not, collect logs of the corresponding component for troubleshooting.
 
+6. Check whether PD has configured `placement-rule` for tables.
+
+    Run the `curl http://<pd-ip>:<pd-port>/pd/api/v1/config/rules/group/tiflash` command to view  all TiFlash placement rules on the current PD. If a rule with the ID being `table-<table_id>-r` is found, the PD has configured a placement rule successfully.
```

Comment:
Duplicate spaces

Reply:
Do you mean the spaces before this sentence? This is by design and is correct in markdown format:
![image](https://user-images.githubusercontent.com/89768198/220050900-69384de1-48c4-45ee-9985-4318ec39cb75.png)